### PR TITLE
G Suite: Import constants from Calypso Products package directly

### PR DESCRIPTION
**Pull request by stephanethomas**
This pull request removes product constants defined in `client/lib/gsuite/constants.js`. Those constants were actually coming from the `@automattic/calypso-products` package, and I don't see a good reason for not referencing them directly. Actually, there is a [marginal gain](https://github.com/Automattic/wp-calypso/pull/62203#issuecomment-1077394932) in term of bundles size in doing so.

#### Testing instructions

1. Run `git checkout update/google-workspace-constants` and start your server, or access a [live branch](https://calypso.live/?image=registry.a8c.com/calypso/app:build-29925)
2. Log into a WordPress.com account with a domain but no emails
3. Open the [`Emails` page](http://calypso.localhost:3000/email)
4. Purchase a new Google Workspace account
5. Assert that this account was provisioned successfully
6. Navigate to the [`Purchases` page](http://calypso.localhost:3000/purchases/subscriptions)
7. Try to cancel this account, and assert that the cancelation survey works as before

### CODE REVIEW

File: client/lib/cart-values/cart-items.ts [hunk 1]
```diff
@@ -11,40 +21,28 @@ import {
...
-	isMonthlyProduct,
-	isBiennially,
-	isRenewable,
-	getTermDuration,
-	getPlan,
-	isBloggerPlan,
-	isWpComFreePlan,
 	isWpComBloggerPlan,
-	isDIFMProduct,
+	isWpComFreePlan,
```
Comment (stephanethomas) [inline client/lib/cart-values/cart-items.ts:39]: I've just reordered those imports, and of course added the two from `calypso/lib/gsuite/constants`.

File: client/lib/gsuite/constants.js [hunk 2]
```diff
@@ -1,24 +1,3 @@
...
-	GSUITE_BASIC_SLUG,
-	GSUITE_BUSINESS_SLUG,
-	GSUITE_EXTRA_LICENSE_SLUG,
-};
-
-export const GSUITE_SLUG_PROP_TYPES = PropTypes.oneOf( [
-	GOOGLE_WORKSPACE_BUSINESS_STARTER_YEARLY,
-	GSUITE_BASIC_SLUG,
-	GSUITE_BUSINESS_SLUG,
-] );
```
Comment (stephanethomas) [inline client/lib/gsuite/constants.js:20]: This constant was no longer used.

File: client/lib/cart-values/cart-items.ts (hunk 1 above)
Comment (AllTerrainDeveloper) [reply client/lib/cart-values/cart-items.ts:39]: Nice ordering

**Review by AllTerrainDeveloper — APPROVED** [review]
I'm glad to see that we are still following one of the rules from the [Automattic Creed](https://automattic.com/creed/wont-just-work/) ❤️ 

Everything looks good to me!

**Evidence of Testing**

Provisioned email account:
image

Survey:
image

Successfully removed:
image

**Review by sirbrillig — APPROVED** [review]
Thank you! This was left-over from the creation of the `calypso-products` package, where tons of constants (needed by code that lives outside calypso) were migrated; IIRC the re-exports were added during that process to simplify the number of changes that had to be made, but this is the right way to go.

**Pull request merged**